plugin/pkg/scheduler/factory/factory.go

```diff
@@ -605,6 +621,56 @@ func (c *ConfigFactory) deleteNodeFromCache(obj interface{}) {
 	}
 }
 
+func (c *ConfigFactory) addPDBToCache(obj interface{}) {
```

Comment:
I think it's `configFactory`. Seems you need a rebase of latest change?

Reply:
Yes. Just fixed it.